src/Security/Member.php

```diff
@@ -391,33 +393,6 @@ public function isLockedOut()
     }
```

Comment:
First of all, I dislike the fact that a can* method returns a validationresult, and not a boolean, as per the convention we've set across the framework.

Secondly, I agree highly with @robbieaverill . canLogin() should only be used when trying to login, but isLockedOut() could be used in many other cases (e.g. an admin dashboard for a list of users), or a summary_fields() column in a group member table.

Making it protected just because we don't currently use it outside of that class isn't a very compelling reason. If it stands alone as a unit-testable method, and isn't used solely as a "can login" synonym (which it is not) then it should be kept public.